docs/_data/nav.json5

```diff
@@ -61,6 +61,9 @@
         children: [
             { label: '<hx-accordion>', path: 'hx-accordion' },
             { label: '<hx-accordion-panel>', path: 'hx-accordion-panel' },
+            { label: '<hx-alert>', path: 'hx-alert' },
+            { label: '<hx-alert-action>', path: 'hx-alert-action' },
+            { label: '<hx-alert-description>', path: 'hx-alert-description' },
```

Comment:
`<hx-alert-description>` may be better named as `<hx-alert-body>` to closer match the naming in other components like panels, modals, etc.

Reply:
That naming seems better to me, maybe we can make a similar change to `hx-alert-action`?

I'm not sure what else it could be named, so I'll leave it as is for now. I've made the change for `description -> body` though.